cw-orch/src/daemon/tx_builder.rs

```diff
@@ -122,16 +123,19 @@ impl TxBuilder {
             sequence
         );
 
-        let auth_info = SignerInfo::single_direct(Some(wallet.private_key.public_key()), sequence)
-            .auth_info(fee);
+        let auth_info = SignerInfo {
+            public_key: wallet.private_key.get_signer_public_key(&wallet.secp),
+            mode_info: ModeInfo::single(SignMode::Direct),
```

Comment:
Have you checked out `SignMode::Eip191`? Could possibly do the signing that's currently implemented in `proto/injective.rs`

Reply:
Will try that

Reply:
It doesn't seem to work either

Reply:
okey, np, just wanted to check.